src/pages/heedongChoi/Login/LoginHee.js

```diff
@@ -1,7 +1,87 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import '../../heedongChoi/Login/LoginHee.scss';
```

Comment:
```suggestion
import './LoginHee.scss';
```

같은 경로 안에 있다면 위와 같은 경로로 수정이 가능합니다

Reply:
경로 수정 했습니다!